app.json

```diff
@@ -3,11 +3,10 @@
     "name": "bloompub-viewer-RN",
     "slug": "bloompub-viewer-RN",
     "version": "1.0.0",
-    "orientation": "portrait",
     "icon": "./assets/icon.png",
     "userInterfaceStyle": "light",
     "splash": {
-      "image": "./assets/splash.png",
+      "image": "./assets/icon.png",
```

Comment:
probably ok to delete any unused starter icons now?

Reply:
Done